static/js/dashboard.js

```diff
@@ -520,43 +448,108 @@ function updateTimeRangeForPanel(panelIndex) {
 }
 
 
+// Event listener for Gridstack resize and drag events
+grid.on('change', function(event, items) {
+    items.forEach(function(item) {
+        // Find the panel in localPanels array using its ID
+        let panelIndex = localPanels.findIndex(panel => panel.panelId === item.el.id);
+        if (panelIndex !== -1) {
+            // Update the position and size of the panel in localPanels array
+            localPanels[panelIndex].gridpos.x = item.y;
+            localPanels[panelIndex].gridpos.y = item.x;
```

Comment:
we are swapping the x, y, is this intentional or a typo ?

Reply:
Yeah, that is intentional.